libs/testing/src/ee/ee.organization.service.ts

```diff
@@ -1,11 +1,16 @@
 import { ApiServiceLevelEnum, MemberRoleEnum, MemberStatusEnum } from '@novu/shared';
 import { CommunityOrganizationRepository, MemberRepository, OrganizationRepository } from '@novu/dal';
-import { getEERepository } from './ee.repository.factory';
+import { getEEMemberRepository, getEEOrganizationRepository, getEERepository } from './ee.repository.factory';
 
 export class EEOrganizationService {
-  private organizationRepository = getEERepository<OrganizationRepository>('OrganizationRepository');
+  private organizationRepository: OrganizationRepository;
   private communityOrganizationRepository = new CommunityOrganizationRepository();
-  private memberRepository = getEERepository<MemberRepository>('MemberRepository');
+  private memberRepository: MemberRepository;
+
+  constructor({ mockClerkClient = true }: { mockClerkClient?: boolean } = {}) {
```

Comment:
For the Dashboard E2E tests we don't want to mock the Clerk client, because we initialize the Clerk session with a real user that has the org and metadata that is used by the Dashboard code.